src/seedu/addressbook/Main.java

```diff
@@ -104,14 +104,18 @@ private void recordResult(CommandResult result) {
      * @return result of the command
      */
     private CommandResult executeCommand(Command command)  {
+        if(!storage.exist()){
+        	ui.showToUser(storage.getWarning());
+        }
```

Comment:
It's good that your program does not stop the file from being automatically recovered when the save command is executed. However, the point of this exercise is to practice using try and catches to handle exceptions. Granted that the exercise isn't perfect, practice using try/catch blocks to handle exceptions where you can in future activities.